failed to run `make lint`

## Actual Behavior

```
$ make lint
/bin/sh: 7: Syntax error: end of file unexpected (expecting "fi")
make: *** [Makefile:97: verify-linter-version] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -100,7 +100,7 @@ verify-linter-version:
 	  echo "[!] Yours:  $(INSTALLED_LINT_VERSION)"; \
 		echo "[!] Theirs: $(GH_LINT_VERSION)"; \
 		echo "[!] Upgrade: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin $(GH_LINT_VERSION)"; \
-                GOLANGCI_LINT=$(go env GOPATH)/bin/$(GOLANGCI_LINT)
+                GOLANGCI_LINT=$(go env GOPATH)/bin/$(GOLANGCI_LINT) \
 	  sleep 3; \
 	fi;
 

